jekyll/_cci2/creating-orbs.md

```diff
@@ -218,6 +217,91 @@ workflows:
           greeting_name: world
 ```
 
+## Providing usage examples of orbs
+_The `examples` stanza is available in configuration version 2.1 and later_
+
+As an author of an orb, you may want to document examples of using it in a CircleCI config file, not only to provide a starting point for new users, but also to demonstrate more complicated use cases.
+
+### Simple examples
+Given the following orb:
```

Comment:
make this a complete sentence that includes the fragment below.

Reply:
Fixed